augly/image/functional.py

```diff
@@ -1786,3 +1786,64 @@ def vflip(
     imutils.get_metadata(metadata=metadata, function_name="vflip", **func_kwargs)
 
     return imutils.ret_and_save_image(aug_image, output_path)
+
+
+def skew(
+    image: Union[str, Image.Image],
+    output_path: Optional[str] = None,
+    metadata: Optional[List[Dict[str, Any]]] = None,
+    level: float, 
+    max_level,
+    axis: str
```

Comment:
```suggestion
    image: Union[str, Image.Image],
    output_path: Optional[str] = None,
    level: float = 5.0,
    max_level: float = 10.0,
    axis: str = "x_axis",
    metadata: Optional[List[Dict[str, Any]]] = None,
```

Reply:
Note: as in all other augmentations, image & output_path must be first & metadata must be last. Also, all kwargs should have default values. Please choose values you think are sensible; I just put placeholder values here.